x-pack/test/functional/apps/discover/group2/feature_controls/discover_security.ts

```diff
@@ -131,10 +141,9 @@ export default function (ctx: FtrProviderContext) {
       });
 
       it('shows CSV reports', async () => {
-        await share.clickShareTopNavButton();
-        await share.clickTab('Export');
+        await exports.clickExportTopNavButton();
```

Comment:
Can we also add a test to this test suite to check that "Export" button is not present when there are not enough privileges?

Reply:
We have unit tests already ascertaining that if we compute there are no available integrations the export button does not render, I think adding such test into FTR would require accounting for every export integration ever registered instead as opposed to delegating this to the publisher of each export integration.